cmd/server/flags.go

```diff
@@ -58,6 +58,12 @@ var flags = []cli.Flag{
 		Name:   "lets-encrypt",
 		Usage:  "enable let's encrypt",
 	},
+	cli.StringFlag{
```

Comment:
this is breaking so I'm not sure if we should drop `WOODPECKER_LETS_ENCRYPT` and enable if `len(...EMAIL) != 0`?

Reply:
That's an idea. But on the other hand it could be a bit unintuitive for the user that setting the let's encrypt email means enabling.

Reply:
ok let it stay as is ...